src/phases/egg-lapse-phase.ts

```diff
@@ -1,11 +1,18 @@
 import BattleScene from "#app/battle-scene.js";
-import { Egg } from "#app/data/egg.js";
+import { Egg, EGG_SEED } from "#app/data/egg.js";
```

Comment:
```suggestion
import { Egg, EGG_SEED } from "#app/data/egg";
```
NIT: Same comment as above